lib/readline4.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/lib/readline4" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>GNU Readline Library</name>
+  <summary xml:lang="en">Get a line from a user with editing.</summary>
+  <description xml:lang="en">Readline offers editing capabilities while the user is entering the line. By default, the line editing commands are similar to those of emacs.</description>
+  <icon href="http://0install.de/feed-icons/Gnu.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="http://0install.de/feed-icons/Gnu.png" type="image/png"/>
```

Comment:
I've been phasing out the use of `http://0install.de/feed-icons/`. If you wish to keep the Gnu icon, better replace these lines with:
```xml
  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png"/>
```

Reply:
replaced